components/send/LedgerSending.tsx

```diff
@@ -57,12 +57,6 @@ export default function LedgerSending({
       }
 
       const account = accountFactory.createFromLedger(transport, accountIndex);
-      const publicKeys = await account.getPublicKeys();
-
-      if (secret.value !== publicKeys[0]) {
-        throw new Error("Invalid Ledger device");
-      }
```

Comment:
This makes user cannot sign transaction for other accounts whose index is larger than 0.